build.gradle

```diff
@@ -1,81 +1,1103 @@
+//version: 1667597057
```

Comment:
This is not the newest version, run the gradle task `updateBuildScript` to update

Reply:
Fixed